hooks/post_gen_project.py

```diff
@@ -210,10 +214,7 @@ def open_pr(source_repo, working_dir):
         if collaborators:
             terraform_vars['access_teams'] = collaborators
 
-        with open(directory + "terraform.tfvars.json", "w") as fh_:
-            fh_.write(json.dumps(terraform_vars))
-
-        run_terraform(directory, TF_MODULE_URL)
+        run_terraform(directory, terraform_vars,  TF_MODULE_URL)
```

Comment:
I can see this question is unrelated to the actual... but if `run_terraform` raises, then the directory hangs around. Might make sense to put a try/finally around this to manage the cleanup.

```suggestion
        try:
            run_terraform(directory, terraform_vars,  TF_MODULE_URL)
        finally:
            shutil.rmtree(os.path.join(os.getcwd(), directory))
```

Reply:
Not needed. Cookiecutter automatically deletes the entire thing if the hook returns a non-zero exit code

Reply:
roger, that works